src/lib/api/search.ts

```
@@ -0,0 +1,181 @@
+    const trimmed = typeof query === "string" ? query.trim() : "";
+
+    let results: FinnhubSearchResult[] = [];
+
+    if (!trimmed) {
+      // Fetch top 10 popular symbols' profiles
+      const top = POPULAR_STOCK_SYMBOLS.slice(0, 10);
+      const profiles = await Promise.all(
+        top.map(async (sym) => {
+          try {
+            const url = `${baseUrl}/stock/profile2?symbol=${encodeURIComponent(sym)}&token=${token}`;
+            const profile = await fetchJSON<any>(url, 3600); // 1h
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)